src/testing/test-utils.ts

```diff
@@ -1,5 +1,39 @@
-import { act, renderHook, RenderHookOptions, RenderHookResult, within } from '@testing-library/react';
+import { Theme, ThemeProvider } from '@terra-ui-packages/components';
+import {
+  act,
+  render,
+  renderHook,
+  RenderHookOptions,
+  RenderHookResult,
+  RenderOptions,
+  within,
+} from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
+import { PropsWithChildren, ReactElement } from 'react';
+import { h } from 'react-hyperscript-helpers';
+
+const testTheme: Theme = {
+  colorPalette: {
+    primary: '#74ae43',
+    secondary: '#6d6e70',
+    accent: '#4d72aa',
+    success: '#74ae43',
+    warning: '#f7981c',
+    danger: '#db3214',
+    light: '#e9ecef',
+    dark: '#333f52',
+    grey: '#808080',
+    disabled: '#b6b7b8',
+  },
+};
+
+const AppProviders = ({ children }: PropsWithChildren<{}>): ReactElement => {
+  return h(ThemeProvider, { theme: testTheme }, [children]);
```

Comment:
hmm.... I think at a minimum, we should move this somewhere else... I was gonna so move it to the components package since that is where knowledge of Theme is comming from...  but on second thought, I think if the goal is to grow this to easy-path multiple app contexts for testing, it should live back over in Terra UI... a test-utils there.

Reply:
Right, if this is _app_ utils, I imagine it will expand to include other Contexts used by the app (LocationContext, etc). It is currently located in the test-utils module in the Terra UI app.

Reply:
lol, my bad.... I should have looked a the file path... it already IS in terra-ui app.